### PR TITLE
Quiz 2

### PR DESCRIPTION
**Pull request by adambray**

### CODE REVIEW

File: quizzes/quiz2_answers.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+* Selects all li's which are anywhere inside a ul of class dropdown (any ancestor)
+* Selects all ul's of class dropdown, as well as the children elements that are li's
+* Selects all ul's of class dropdown, only if their children are exclusively li's
+```
+
+## Scope/Context/Closures
+
+### Question
+
+Describe the rules of scope in JavaScript.
```
Comment (RobertAKARobin) [inline quizzes/quiz2_answers.md:24]: Maybe "Describe two rules of scope in Javascript"

File: quizzes/quiz2_answers.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+### Question
+
+Define an object and store it in a variable `pizza`. The object should have 2
+properties: a temperature (set to 70), and a method called `bake`. When called,
```
Comment (RobertAKARobin) [inline quizzes/quiz2_answers.md:29]: "a temperature of 70"

File: quizzes/quiz2_answers.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+this method should set the pizza's temperature to be 300. Note: you may not use
+the variable pizza inside your method.
```
Comment (RobertAKARobin) [inline quizzes/quiz2_answers.md:31]: the variable `pizza`

File: quizzes/quiz2_answers.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+
+## Callbacks
+
+### Question
+
+**Define a function called `doSomething`. It should take one argument, called
+`thingToDo`. When called, thing to do should invoke the function given as an
```
Comment (RobertAKARobin) [inline quizzes/quiz2_answers.md:38]: When called, `thingtodo`

File: quizzes/quiz2_answers.md [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+* fork on github; git clone <ga_dc_url>; git checkout -b <charlie_solution>; git add <files>; git commit; git push; create pull request
+* git clone <ga_dc_url>; git branch <charlie_solution>; git add <files>; git commit; git push; create pull request
+* fork on github; git clone <fork_url>; git checkout -b <charlie_solution>; git add <files>; git commit; git pull; create pull request
+```
+
+## jQuery
+
+### Question
+
+Which of the following statements will work, assuming jQuery is loaded? SELECT ALL THAT APPLY
```
Comment (RobertAKARobin) [inline quizzes/quiz2_answers.md:69]: Which "will work", or which are correct? `.innerHTML` will work, but shouldn't be used.

Comment (adambray) [inline quizzes/quiz2_answers.md:69]: `.innerHTML` doesn't work for me. Are you sure about that?

**Pull request closed without merging**